modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1107,72 +1137,167 @@ Type::
 |===
 | Property | Type | Description
 
-| `authToken`
-| `string`
-| `authToken` describes the way to get a token to authenticate to Loki. +
- - `DISABLED` does not send any token with the request. +
- - `FORWARD` forwards the user token for authorization. +
- - `HOST` [deprecated (*)] - uses the local pod service account to authenticate to Loki. +
- When using the Loki Operator, this must be set to `FORWARD`.
+| `advanced`
+| `object`
+| `advanced` allows setting some aspects of the internal configuration of the Loki clients. This section is aimed mostly for debugging and fine-grained performance optimizations.
 
-| `batchSize`
-| `integer`
-| `batchSize` is the maximum batch size (in bytes) of logs to accumulate before sending.
+| `enable`
+| `boolean`
+| Set `enable` to `true` to store flows in Loki. It is required for the {product-title} Console plugin installation.
+
+| `lokiStack`
+| `object`
+| Loki configuration for "LokiStack" mode. This is useful for an easy loki-operator configuration. It is ignored for other modes.
```

Comment:
Same for this mode. I think if we mention it, should it be in ` ` and also match the casing that a user would need to use in the YAML? Like `lokiStack`?

Reply:
no the case here is important it's "LokiStack" for the value of `mode` ; and there's a key named `lokiStack`. I think it's good as it is